wls-gui-wahllokalsystem/tests/components/vorfaelleundvorkommnisse/TheEreignisseNoEventsCheckboxes.spec.ts

```diff
@@ -172,5 +174,24 @@ describe("TheEreignisseNoEventsCheckboxes.vue", () => {
         ereignisStore.wahlbezirkEreignisse.keineVorkommnisse
       ).toStrictEqual(true);
     });
+
+    it("should_updateKeineVorkommnisseToTrue_when_keineVorkommnisseCheckboxWasSelectedForBWB", async () => {
+      const ereignisStore = useEreignisStore();
+      const user = prepareUser().wahlbezirksArt(WahlbezirksArtEnum.BWB).build();
+      useUserStore().setUser(user);
```

Comment:
die extra variable wird nicht gebraucht
```suggestion
      useUserStore().setUser(prepareUser().wahlbezirksArt(WahlbezirksArtEnum.BWB).build());
```